helm/ffc-pay-statement-generator/values.yaml

```diff
@@ -40,7 +40,10 @@ container:
   storageContainer: statements
   storageFolder: outbound
   statementVersion: v1
-  statementReceiverEndpoint: http://ffc-pay-statement-receiver.ffc-pay
+  statementReceiverEndpoint: https://ffc-pay-statement-receiver
+
+ingress:
+  server: example.com
```

Comment:
Using `ingress` object will mean existing app config will populate this value in each environment without the need to manage additional duplicate configuration.